CMakeLists.txt

```diff
@@ -40,55 +40,55 @@ endif()
 # MEMCHECK ==> LIBCELLML_MEMCHECK
 set(_PARAM_ANNOTATION "Enable memcheck testing.")
 if(VALGRIND_TESTING_AVAILABLE)
-  set( LIBCELLML_MEMCHECK ON CACHE BOOL ${_PARAM_ANNOTATION} )
+  set(LIBCELLML_MEMCHECK ON CACHE BOOL ${_PARAM_ANNOTATION})
 endif()
-if( MEMCHECK AND VALGRIND_TESTING_AVAILABLE )
-  set( LIBCELLML_MEMCHECK "${MEMCHECK}" CACHE BOOL ${_PARAM_ANNOTATION} FORCE )
+if(MEMCHECK AND VALGRIND_TESTING_AVAILABLE)
+  set(LIBCELLML_MEMCHECK "${MEMCHECK}" CACHE BOOL ${_PARAM_ANNOTATION} FORCE)
 elseif(MEMCHECK)
   message(WARNING "Memcheck testing requested but valgrind or its requirements not found!")
   unset(MEMCHECK CACHE)
 endif()
 
 # UNIT_TESTS ==> LIBCELLML_UNIT_TESTS
 set(_PARAM_ANNOTATION "Enable libCellML tests.")
-set( LIBCELLML_UNIT_TESTS ON CACHE BOOL ${_PARAM_ANNOTATION} )
-if( UNIT_TESTS )
-  set( LIBCELLML_UNIT_TESTS "${UNIT_TESTS}" CACHE BOOL ${_PARAM_ANNOTATION} FORCE )
+set(LIBCELLML_UNIT_TESTS ON CACHE BOOL ${_PARAM_ANNOTATION})
+if(UNIT_TESTS)
+  set(LIBCELLML_UNIT_TESTS "${UNIT_TESTS}" CACHE BOOL ${_PARAM_ANNOTATION} FORCE)
 endif()
 
 if(LIBCELLML_COVERAGE OR LIBCELLML_MEMCHECK)
   set(LIBCELLML_UNIT_TESTS ON CACHE BOOL ${_PARAM_ANNOTATION} FORCE)
 endif()
 
-# TWAE ==> LIBCELLML_WARNINGS_AS_ERRORS -- Note: This excludes third party code, where warnings are never treated as errors.
+# TREAT_WARNINGS_AS_ERRORS ==> LIBCELLML_WARNINGS_AS_ERRORS -- Note: This excludes third party code, where warnings are never treated as errors.
```

Comment:
It's not about looking nice, but about being consistent. Anyway, I have shortened it a bit. Hopefully, you find it more usable that way (not that this option should be used that often anyway).